Makefile

```diff
@@ -0,0 +1,16 @@
+NAME    := libcircle
+SRC_EXT := gz
+DL_VERSION = $(VERSION)-rc.1
+SOURCE = https://github.com/hpc/$(NAME)/releases/download/$(DL_VERSION)/$(NAME)-$(DL_VERSION).tar.$(SRC_EXT)
```

Comment:
This should come from the spec file, so that we eliminate multiple-update-locations problems.